### ISSUE TITLE
Exit proccess broken

### ISSUE THREAD

**Issue by joaop221** [joaop221/v-rising-docker-arm64#1]
Error:

```bash
/home/steam/init-server.sh: line 25: kill: 145
147: arguments must be process or job IDs
/home/steam/init-server.sh: line 26: wait: `145
147': not a pid or valid job spec
```

**Comment by joaop221** [comment]
fixed in [2292b32](https://github.com/joaop221/v-rising-docker-arm64/commit/2292b323965ec16496a32adf9fd3a44e40545bf2)

**Comment by joaop221** [comment]
fixed in [2292b32](https://github.com/joaop221/v-rising-docker-arm64/commit/2292b323965ec16496a32adf9fd3a44e40545bf2)
